Debug App/Sources/View Controllers/Merchant Helpers/MerchantHelpers.swift

```diff
@@ -0,0 +1,143 @@
+                postalCode: "10178")
+        ),
+        order: ClientSessionRequestBody.Order(
+            countryCode: .de,
+            lineItems: [
+                ClientSessionRequestBody.Order.LineItem(
+                    itemId: "item-123",
+                    description: "this item",
+                    amount: 1000,
+                    quantity: 1,
+                    discountAmount: nil,
+                    taxAmount: nil,
+                    productType: "DIGITAL")
+            ]),
+        paymentMethod: klarnaPaymentMethod,
+        testParams: nil)
+    
+    static var normalClientSession = ClientSessionRequestBody(
+        customerId: customerId,
+        orderId: "ios-order-\(String.randomString(length: 8))",
+        currencyCode: CurrencyLoader().getCurrency("EUR"),
+        amount: nil,
+        metadata: nil,
+        customer: ClientSessionRequestBody.Customer(
+            firstName: "John",
+            lastName: "Smith",
+            emailAddress: "john@primer.io",
+            mobileNumber: "+4478888888888",
+            billingAddress: Address(
+                firstName: "John",
+                lastName: "Smith",
+                addressLine1: "65 York Road",
+                addressLine2: nil,
+                city: "London",
+                state: "Greater London",
+                countryCode: "GB",
+                postalCode: "NW06 4OM"),
+            shippingAddress: Address(
+                firstName: "John",
+                lastName: "Smith",
+                addressLine1: "9446 Richmond Road",
+                addressLine2: nil,
+                city: "London",
+                state: "Greater London",
+                countryCode: "GB",
+                postalCode: "EC53 8BT")
+        ),
+        order: ClientSessionRequestBody.Order(
+            countryCode: .de,
+            lineItems: [
+                ClientSessionRequestBody.Order.LineItem(
+                    itemId: "fancy-shoes-\(String.randomString(length: 4))",
+                    description: "Fancy Shoes",
+                    amount: 600,
+                    quantity: 1,
+                    discountAmount: nil,
+                    taxAmount: nil)
+            ]),
+        paymentMethod: normalPaymentMethod,
+        testParams: nil)
+    
+    static var normalPaymentMethod = ClientSessionRequestBody.PaymentMethod(
+        vaultOnSuccess: false,
+        options: nil,
+        descriptor: nil,
+        paymentType: nil
+    )
+    
+    static var klarnaPaymentMethod = ClientSessionRequestBody.PaymentMethod(
+        vaultOnSuccess: nil,
+        options: paymentOptions,
+        descriptor: "test-descriptor",
+        paymentType: nil
+    )
+    
+    static var paymentOptions = ClientSessionRequestBody.PaymentMethod.PaymentMethodOptionGroup(
+        KLARNA: ClientSessionRequestBody.PaymentMethod.PaymentMethodOption(
+            surcharge: ClientSessionRequestBody.PaymentMethod.SurchargeOption(amount: 140)))
```

Comment:
When is this being used? The hardcoded surcharge amount stands out as odd

Reply:
This is copied from Android, they have hardcoded all the payment options for all the payment methods. I only hardcoded the value for Klarna so the BE will return us in `https://sdk.api.staging.primer.io/client-session/actions` request `fees` values.

These fees (if we sent any) are added as `orderItems` in the body for the request: `https://api.staging.primer.io/klarna/payment-sessions` like this:

`"orderItems": [
    {
      "name": "Fancy Shoes",
      "discountAmount": 0,
      "reference": "fancy-shoes-mdqp",
      "quantity": 1,
      "taxAmount": 0,
      "unitAmount": 600
    },
    {
      "unitAmount": 140,
      "name": "surcharge",
      "productType": "surcharge",
      "quantity": 1
    }
  ]`